src/app/View.elm

```diff
@@ -33,6 +34,17 @@ firstPlayer players =
             "No Players Found"
 
 
-tableView : Html Msg
-tableView =
-    Html.map TableMsg (Table.view Table.init)
+tableView : List Player -> Dice.Cup -> Html Msg
+tableView players cup =
+    let
+        -- we have to fake ids until we have them
+        idAndPlayerTo3Tuple id { playerName, diceCount } =
+            ( id, playerName, diceCount )
+
+        ids =
+            List.range 1 <| List.length players
+
+        playersTuples =
+            List.map2 idAndPlayerTo3Tuple ids players
+    in
+        Table.view Types.TableAction <| Table.initWithPlayersAndCup playersTuples cup
```

Comment:
This is the problem with calling this function `Table.initWithPlayersAndCup`. It should just be `Table.fromPlayersAndCup`

The `tableView` function is just a helper function bandaid because we have no player ids in our model in `Types.Player`.  Ideally it'd be in the `Model` (which doesn't exist yet), or a `Utils` or `Helper` module somewhere.

This function is the glue that stitches the main Model's players to the Table for rendering it. 

Notice I'm not mounting anything in the main `Model` for the Table? Currently nothing needs to be persisted, so all it does is call a function with the data it needs.

When / if we change that, we'll worry about that split then... (It'll be interesting then, because we won't be directly driving the model data of the `Table` from the model data of the backend / `Main` module.

This is very much WIP because of that, but I wanted to make sure we can all work on things separately, so that's what this does... separates things out.